zrml/global-disputes/src/lib.rs

```diff
@@ -647,8 +833,88 @@ mod pallet {
             }
         }
 
-        fn is_started(market_id: &MarketIdOf<T>) -> bool {
-            <Winners<T>>::get(market_id).is_some()
+        fn does_exist(market_id: &MarketIdOf<T>) -> bool {
+            <GlobalDisputesInfo<T>>::get(market_id).is_some()
+        }
+
+        fn is_unfinished(market_id: &MarketIdOf<T>) -> bool {
+            if let Some(gd_info) = <GlobalDisputesInfo<T>>::get(market_id) {
+                return matches!(
+                    gd_info.status,
+                    GDStatus::Active { add_outcome_end: _, vote_end: _ } | GDStatus::Initialized
+                );
```

Comment:
Totally agreed. It's confusing. When I delete the `Initialized` state, this will get fixed too. I am in favour of renaming it to `is_active` again. It was like that before, but the initialized state needed a renaming. The reason for the initialized state was the suggestion to start a global dispute as API call and I left the `push_vote_outcome` call around.